html/changelogs/RemieRichards-ExplosionsAndEmbedding.yml

```diff
@@ -0,0 +1,12 @@
+  
+author: RemieRichards
+
+delete-after: True
+
+changes: 
+  - rscadd: "Explosions will now blow (throw) items away from the epicenter"
+  - rscadd: "Sharp items can now embedd themselves in a human's limbs"
```

Comment:
yeah that's an error too, if you've seen I corrected all the "embedd"s to embed where they're visible in game and in the code. I don't believe the changelog will matter but if requested to I'll fix the typo.